### ISSUE TITLE
Error on trilium 0.62.5 because of the api.runOnBackend()

### ISSUE THREAD

**Issue by dariuszszyc** [madodig/trilium-widget-theme-switch#2]
It appears that Trilium 0.62.5 (self-hosted from docker) throws errors because of the 
`api.runOnBackend`
It seems to work fine once the above line is replaced (3x in the lib file) with
`api.runAsyncOnBackendWithManualTransactionHandling`

![msedge_ywPvI2QY4l](https://github.com/madodig/trilium-widget-theme-switch/assets/41972270/c1281197-d2cd-4e29-9625-fd2254615b69)

**Comment by madodig** [comment]
Hi, thanks, I've seen this a week ago when I upgraded my Trilium to 0.62.4. Fixed it, but didn't update the repo. Until I do, I'll leave this open.

For anyone else experiencing this issue, until I release a fixed version, open the 'lib' note (under the ThemeSwitch note) and replace all occurrences of
api.runOnBackend
with 
api.runAsyncOnBackendWithManualTransactionHandling
If I'm not wrong, 3 lines are affected (8, 15, 22).
